utilities-core/src/test/java/uk/gov/justice/services/yaml/YamlParserTest.java

```diff
@@ -0,0 +1,169 @@
+package uk.gov.justice.services.yaml;
+
+import static java.nio.file.Paths.get;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.instanceOf;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.fail;
+import static org.mockito.Mockito.when;
+
+import uk.gov.justice.services.yaml.subscriptiondescriptor.Event;
+import uk.gov.justice.services.yaml.subscriptiondescriptor.Subscription;
+import uk.gov.justice.services.yaml.subscriptiondescriptor.SubscriptionDescriptor;
+import uk.gov.justice.services.yaml.subscriptiondescriptor.SubscriptionDescriptorDef;
+
+import java.io.FileNotFoundException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class YamlParserTest {
```

Comment:
This test class is inconsistent.  One test uses mocks, then the other tests switch to an implementation of the YamlParser.  It should be one or the other or two different test classes for clarity.